lib/actions/local/configure.coffee

```
@@ -95,8 +66,48 @@ inquirerOptions = (data) -> [
 	}
 ]
 
+# Since these paths are used by `resin-image-fs`, we can safely
+# assume they are always UNIX style.
+joinPath = (path) -> '/' + path.join('/')
+
+loadReconfix = (target) ->
+	{ Reconfix } = require('reconfix-preview')
+	imagefs = require('resin-image-fs')
+
+	reconfix = new Reconfix({
+		read: (partition, path) ->
+			imagefs.readFile
+				image: target
+				partition: partition
+				path: joinPath(path)
+			.catch (err) ->
+				if err.code == 'NOENT'
+					return Buffer.alloc(0)
+				else
+					throw err
```

Why do we throw an error here if we're going to swallow it two lines later?